media-sound/jack-audio-connection-kit/jack-audio-connection-kit-0.125.0.ebuild

```diff
@@ -57,7 +57,6 @@ multilib_src_configure() {
 		--disable-firewire \
 		$(use_enable cpu_flags_x86_sse sse) \
 		--with-html-dir=/usr/share/doc/${PF} \
-		--disable-dependency-tracking \
 		--libdir=/usr/$(get_libdir) \
```

Comment:
`--libdir` is also the default, isn't it?
And after removing the line, inherit multilib could be dropped, I think.